images/linux/scripts/tests/Haskell.Tests.ps1

```diff
@@ -1,22 +1,5 @@
 Describe "Haskell" {
 
```

Comment:
@hasufell , Could you please revert and fix these tests? We need to be sure that we have 2 version have been installed and we can get --version output for them.

- GHC directory contains two version of GHC
- GHC version